docs/index.rst

```diff
@@ -42,9 +42,12 @@ application:
     # config/packages/doctrine_migrations.yaml
 
     doctrine_migrations:
+        # Whether to enable fetching migrations from the service container.
+        enable_service_migrations: false
+
         # List of namespace/path pairs to search for migrations, at least one required
         migrations_paths:
-            'App\Migrations': '%kernel.project_dir%/src/App'
+            'App\Migrations': '%kernel.project_dir%/src/Migrations'
```

Comment:
The directory should not be required when a namespace is specified. Maybe add a `namespaces` configuration, that is an optional list of PHP namespaces or class names. All migrations from the container are used by default.

Reply:
The directory is still required when generating new migrations so the command knows where to put them.